.github/workflows/perf_micro.yml

```diff
@@ -7,6 +7,13 @@ on:
       - 'release/**'
     tags:
       - '**'
+    paths:
+      - 'cmake/**'
+      - 'CMakeLists.txt'
+      - '.github/workflows/perf_micro.yml'
+      - 'perf/**'
+      - 'src/**'
+      - 'third_party/**'
```

Comment:
I would also add `patches/**` here.

Minor: please sort entries alphabetically.

Reply:
> I would also add patches/** here.

Added.

> Minor: please sort entries alphabetically.

What's wrong with a current order?

Reply:
I suppose that `.github/...` should be before `[Cc]make**`, feel free to ignore.